common/src/main/java/com/wynntils/screens/WynntilsQuestBookScreen.java

```diff
@@ -278,21 +279,31 @@ private void renderTooltip(PoseStack poseStack, int mouseX, int mouseY) {
                     tooltipLines,
                     FontRenderer.getInstance().getFont(),
                     true);
+            return;
         }
-    }
 
-    private void renderPageInfo(PoseStack poseStack) {
-        FontRenderer.getInstance()
-                .renderAlignedTextInBox(
-                        poseStack,
-                        (currentPage + 1) + " / " + (maxPage + 1),
-                        Texture.QUEST_BOOK_BACKGROUND.width() / 2f,
-                        Texture.QUEST_BOOK_BACKGROUND.width(),
-                        Texture.QUEST_BOOK_BACKGROUND.height() - 25,
-                        0,
-                        CommonColors.BLACK,
-                        HorizontalAlignment.Center,
-                        FontRenderer.TextShadow.NONE);
+        if (this.hovered instanceof DialogueHistoryButton) {
```

Comment:
I'm not sure why Qodana is complaining about this here but not in `renderTooltip()`. But can you please check if there is some substance behind this claim?

Reply:
I have checked this and tested in-game, and it seems to work fine. I have no idea what Qodana is doing.